src/overlays/actors/ovl_Bg_Spot11_Oasis/z_bg_spot11_oasis.c

```diff
@@ -106,19 +106,19 @@ void func_808B29E0(BgSpot11Oasis* this) {
 
 void func_808B29F0(BgSpot11Oasis* this, PlayState* play) {
     if (Math_StepToF(&this->actor.world.pos.y, 0.0f, 0.7f)) {
-        func_808B2AA8(this);
+        BgSpot11Oasis_SetActionToNothing(this);
         Actor_Spawn(&play->actorCtx, play, ACTOR_EN_ELF, this->actor.world.pos.x, this->actor.world.pos.y + 40.0f,
                     this->actor.world.pos.z, 0, 0, 0, FAIRY_SPAWNER);
         Sfx_PlaySfxCentered(NA_SE_SY_CORRECT_CHIME);
     }
     func_808B27F0(play, this->actor.world.pos.y);
 }
 
-void func_808B2AA8(BgSpot11Oasis* this) {
-    this->actionFunc = func_808B2AB8;
+void BgSpot11Oasis_SetActionToNothing(BgSpot11Oasis* this) {
```

Comment:
The convention seems to be `<Actor>_Setup<Action>`, so `BgSpot11Oasis_SetupDoNothing` (like `BgJya1flift_SetupDoNothing`). You could probably apply this to other functions in this PR too, like `func_808B3274`